README.md

```diff
@@ -31,6 +31,7 @@ I need your help to maintain this list up to date 🙏. See [contribution Guide]
 > Read the story behind this project: [My simple GitHub project went viral 🚀](https://gourav.io/blog/my-simple-github-project-went-viral)
 
 ## Clones with Tutorials
+
```

Comment:
@SamarthKadam Could you kindly remove the extra line you added? It appears that this extra line is causing about 190 lines to show up as changes in the comparison, making it harder to see exactly what modifications were made. Your help in fixing this would be really helpful.

Reply:
@SamarthKadam also resolve the merge conflicts